packages/minter-contracts/ligo/fa2_modules/fa2_allowlist/allowlist_base.mligo

```diff
@@ -0,0 +1,23 @@
+(* Base allowlisting module.
+ * Import it in a contract if it is planned to be extended with allowlisting
+ * extensions.
+*)
+
+#if !ALLOWLIST_BASE
+#define ALLOWLIST_BASE
+
+#if !ALLOWLIST_ENABLED
```

Comment:
I have a question: couldn't you remove `ALLOWLIST_ENABLED` (I mean everywhere) in favor of `ALLOWLIST_BASE`?

Unless I'm missing something it looks like in the contract there are some checks for `#if !ALLOWLIST_ENABLED`, but AFAIU if no allowlist is enabled this module still needs to be imported, so `ALLOWLIST_BASE` will be defined.

Note: if you do so however in `#ifelse` chains I suppose the `!ALLOWLIST_ENABLED` that is now on top should be replaced by an `ALLOWLIST_BASE` at the bottom.

Reply:
`ALLOWLIST_ENABLED` in the first place serves to notify `allowlist_base.mligo` that some allowlisting implementation has been included and providing primitives for empty allowlisting is not necessary.

Though, from other modules' point of view, maybe the existence of `ALLOWLIST_ENABLED` complicates things. Do you think it's worth rewriting via `ALLOWLIST_BASE`?